cli/src/prepareProject.ts

```diff
@@ -28,13 +28,21 @@ Either set this project up as a Git repo, or configure the website title in moon
 
 export type FoldersEnabled = { [index in typeof COPY_FOLDERS[number]]: boolean }
 
+export type ClassOrder = (
+  | string
+  | {
+      section?: string
+      classes: string[]
+    }
+)[]
+
```

Comment:
After the change to allow simple names, this type was wrong since it could be an array of strings as well. I also changed this type so that the array is part of the type alias instead of the type alias referring to a single element of the array.